rotkehlchen/chain/ethereum/modules/makerdao/decoder.py

```diff
@@ -509,6 +577,7 @@ def addresses_to_decoders(self) -> dict[ChecksumEvmAddress, tuple[Any, ...]]:
             string_to_evm_address('0xA26e15C895EFc0616177B7c1e7270A4C7D51C997'): (self.decode_proxy_creation,),  # noqa: E501
             string_to_evm_address('0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359'): (self.decode_saidai_migration,),  # noqa: E501
             self.makerdao_cdp_manager.address: (self.decode_cdp_manager_events,),
+            string_to_evm_address('0x83F20F44975D03b1b09e64809B757c47f942BEeA'): (self.decode_sdai_events,),  # noqa: E501
         }
 
     def counterparties(self) -> list[CounterpartyDetails]:
```

Comment:
Here you need to add the full details for the sdai counterparty.

Use this logo
![sdai](https://github.com/rotki/rotki/assets/1658405/3dac7070-9b64-4d1b-9214-5a2133612975)


We maybe should change icon logo of sdai to this too but coingecko which is where we get it from does not have a different one.